util/src/main/resources/org/killbill/billing/util/security/shiro/dao/JDBCSessionSqlDao.sql.stg

```diff
@@ -45,4 +45,19 @@ delete() ::= <<
 delete from sessions
 where id = :id
 ;
->>
\ No newline at end of file
+>>
+
+findOldActiveSessions() ::= <<
+select
+  record_id
+, id
+, start_timestamp
+, last_access_time
+, timeout
+, host
+, session_data
+from sessions
```

Comment:
I was expecting to see a where close that filters the *old* sessions ?

Reply:
The check for `old` (orphaned) is done in the code. The `sessions` table should almost always be empty, except for currently active sessions (Kaui users).

Reply:
Nit: In that case renaming that query `findOldActiveSessions` -> `findActiveSessions` would make sense

Reply:
Not quite, it's really `grabASubsetOfTheSessionsOrderedByCreatedDateAndCheckIfTheyAreOrphaned`... Naming is hard 😖